lib/Service/ETLService.php

```diff
@@ -120,23 +120,19 @@ public function extractAndTransform(int $max = self::MAX_BATCH_SIZE, OutputInter
 			->where($select->expr()->eq('id', $select->createParameter('id')));
 
 		foreach ($this->getRaw($max, $output) as $row) {
-			$insert->setParameter('uid', $row['uid']);
-			$insert->setParameter('ip', $row['ip']);
-			$insert->setParameter('seen', 1, IQueryBuilder::PARAM_INT);
-			$insert->setParameter('ts', $row['created_at'], IQueryBuilder::PARAM_INT);
-
-			try {
+			$select
+				->setParameter('uid', $row['uid'])
+				->setParameter('ip', $row['ip']);
+			$result = $select->execute();
+			$existing = $result->fetch();
+			$result->closeCursor();
```

Comment:
After running tests and analysing others code, I found that there was `closeCursor()` statement